### PR TITLE
docs: update tagging policy docs to describe individual tag resources

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

File: docs/infracost_cloud/tagging_policies.md [hunk 1]
```diff
@@ -78,12 +78,17 @@ If you do not use source control integrations ([GitHub App](/docs/integrations/g
...
 - For tags set in modules, the actual module version being used is checked.
-- For `aws_instance` with `ebs_block_device` or `root_block_device` definitions, tags for the attached volumes are checked as follows:
-  - if `volume_tags` attribute is set it is checked. Otherwise,
-  - if there is at least one `ebs_block_device` and no `*_block_device.tags` set, `volume_tags` are checked. Otherwise,
-  - `.tags` for each `*_block_device` are checked.
-- For `aws_launch_template`, the `tag_specifications` attribute is also checked. If the `resource_type` is `instance` or `volume` these tags are then associated with either the `aws_instance` or `aws_autoscaling_group` resource that references the `aws_launch_template` and checked as part of those resources.
+
+
+<details><summary>AWS-specific notes</summary>
+<ul><li>For <code>aws_autoscaling_group</code>, if the <code>propagate_at_launch</code> attribute is not set to true, the resource fails tagging policies as resources launched from those Auto Scaling groups will not get the required tags.</li>
```
Comment (alikhajeh1) [inline docs/infracost_cloud/tagging_policies.md:85]: @aliscott do you know if there's a way to use markdown syntax inside html blocks? that would make this code more readable but it's not a big issue.

Comment (aliscott) [reply docs/infracost_cloud/tagging_policies.md:85]: Adding empty lines might work ([see here](https://docusaurus.io/docs/markdown-features/react#markdown-and-jsx-interoperability)).

e.g.:
```html
<ul>
  <li>
  
For `aws_autoscaling_group`, if the ...

  </li>
</ul>
```

File: docs/infracost_cloud/tagging_policies.md [hunk 2]
```diff
@@ -78,12 +78,17 @@ If you do not use source control integrations ([GitHub App](/docs/integrations/g
...
+<li>For <code>aws_instance</code> with <code>ebs_block_device</code> or <code>root_block_device</code> definitions, tags for the attached volumes are checked as follows:</li>
+  <ol><li>if <code>volume_tags</code> attribute is set it is checked. Otherwise,</li>
+  <li>if there is at least one <code>ebs_block_device</code> and no <code>*_block_device.tags</code> set, <code>volume_tags</code> are checked. Otherwise,</li>
+  <li><code>.tags</code> for each <code>*_block_device</code> are checked.</li></ol>
+<li>For <code>aws_launch_template</code>, the <code>tag_specifications</code> attribute is also checked. If the <code>resource_type</code> is <code>instance</code> or <code>volume</code> these tags are then associated with either the <code>aws_instance</code> or <code>aws_autoscaling_group</code> resource that references the <code>aws_launch_template</code> and checked as part of those resources.</li>
+<li>The following individual tag resources are not checked as these are used to tag resources defined outside of Terraform: <code>aws_autoscaling_group_tag</code>, <code>aws_ec2_tag</code>, <code>aws_ec2_tag</code>, <code>aws_ecs_tag</code>, <code>aws_ecs_tag</code>.</li></ul>
```
Comment (aliscott) [inline docs/infracost_cloud/tagging_policies.md:91]:
```suggestion
<li>The following individual tag resources are not checked as these are used to tag resources defined outside of Terraform: <code>aws_autoscaling_group_tag</code>, <code>aws_ec2_tag</code>, <code>aws_transfer_tag</code>, <code>aws_ecs_tag</code>, <code>aws_dynamodb_tag</code>.</li></ul>
```

**Review by aliscott — APPROVED** [review]

File: docs/infracost_cloud/tagging_policies.md (hunk 1 above)
Comment (aliscott) [reply docs/infracost_cloud/tagging_policies.md:85]: Or even:

```html
<details>
  <summary>AWS-specific notes</summary>

- For `aws_autoscaling_group`, if the ...

</details>
```

Comment (alikhajeh1) [reply docs/infracost_cloud/tagging_policies.md]: Thanks, empty lines worked!

**Pull request merged**